pkg/proxy/request.go

```diff
@@ -268,6 +286,25 @@ func getDuration(r *http.Request) (time.Duration, error) {
 	return time.ParseDuration(val)
 }
 
+func getImage(r *http.Request, role string) (apisLoadTestV1.ImageDetails, error) {
```

Comment:
maybe you could remove the error return here, as it returns a empty string by default

Reply:
Done!

Reply:
There are some `if err != nil ` checks after the function call, I think those can be removed as well

Reply:
Fixed